pkg/remote/resource_enumeration_error_handler.go

```diff
@@ -56,6 +81,11 @@ func HandleResourceEnumerationError(err error, alerter alerter.AlerterInterface)
 		return handleAWSError(alerter, listError, reqerr)
 	}
 
+	if strings.Contains(rootCause.Error(), "AccessDenied") {
```

Comment:
Can you add as a comment which kind of errors are handled by this statement ? That's not clear for me, that mean that we have error that are not of type `awserr.RequestFailure` for access denied calls ?

Reply:
Ok nevermind, it was just to be sure that this check was not a mistake